index.html

```diff
@@ -12,12 +12,61 @@
   <link rel="stylesheet" href="css/reset.css">
   <link rel="stylesheet" href="css/layout.css">
   <link rel="stylesheet" href="css/styles.css">
+  <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@300;500&display=swap" rel="stylesheet">
 </head>
 <body>
   <header>
+    <h3 class="container">Code Journal</h3>
   </header>
   <main>
     <div data-view="edit-profile"></div>
+    <div class="container">
+      <form id="profile-form">
+        <h2>Edit Profile</h2>
+        <div class="row">
+          <div class="column-half">
+            <img src="images/placeholder-image-square.jpg" alt="avatar" id="avatar">
+          </div>
+          <div class="column-half">
+            <label for="avatar-url">Avatar URL</label>
+            <input
```

Comment:
Your indentation in here is getting a little bit crazy! For the sake of your own sanity, please revise this to follow good HTML code formatting :grin: